packages/scripts/src/upgrade-tests/register-new-contract/1_Propose.ts

```diff
@@ -10,26 +10,27 @@
 // NODE_URL_42161=<ARBITRUM-NODE-URL> \
 // yarn hardhat run ./src/upgrade-tests/register-new-contract/1_Propose.ts --network localhost
 
-const hre = require("hardhat");
-
-const { getAddress } = require("@uma/contracts-node");
-const { RegistryRolesEnum } = require("@uma/common");
 import { BigNumberish } from "@ethersproject/bignumber";
 import { BytesLike } from "@ethersproject/bytes";
+import "@nomiclabs/hardhat-ethers";
```

Comment:
This import is required to fix typescript issues, without it `ts` doesn't allow to do `hre.ethers`